xbmc/music/MusicDatabase.cpp

```diff
@@ -3380,6 +3383,7 @@ bool CMusicDatabase::GetArtistsByWhere(const std::string& strBaseDir, const Filt
 
         pItem->GetMusicInfoTag()->SetDatabaseId(artist.idArtist, MediaTypeArtist);
         pItem->SetIconImage("DefaultArtist.png");
+        pItem->GetMusicInfoTag()->SetArtist(artist);
```

Comment:
Ah it looks like `CFileItem(const CArtist& artist)` could need an improvement to call `CMusicInfoTag::SetArtist(const CArtist& artist)` instead of calling `CMusicInfoTag::SetArtist(const std::string& artist)` which only sets the name of the artist but not the rest of the details.